File: src/test/java/com/tom/StudentTest.java
package com.tom;

import com.tom.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void highestScoreTest(){
        Student student = new Student("Hank",60,80);
        Assertions.assertEquals(80,student.highest());
    }

    @Test
    public void averageTest(){
        Student student = new Student("Hank" , 70 , 70);
        Assertions.assertEquals(70,student.getAverage());
    }
}

File: src/test/kotlin/com/tom/StudentTest.kt
package com.tom

import com.kotlin.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {

    @Test
    fun hightestTest(){
        val student = Student("Hank",78,65)
        Assertions.assertEquals(78,student.highest())
    }
    @Test
    fun averageTest(){
        val student = Student("Hank",80,90)
        Assertions.assertEquals((90+80)/2,student.getAverage())
    }

     @Test
     fun gradingTest(){
         val student = Student("Hank",80,90)
         Assertions.assertEquals('B',student.grading())
     }
}